docs/docsite/rst/user_guide/playbooks_loops.rst

```diff
@@ -1,21 +1,40 @@
 .. _playbooks_loops:
 
+*****
 Loops
-=====
+*****
 
-Often you'll want to do many things in one task, such as create a lot of users, install a lot of packages, or
-repeat a polling step until a certain result is reached.
+Sometimes you want to repeat a task multiple times. In computer programming, this is called a loop. Common Ansible loops include installing several packages with the :ref:`yum module <yum_module>`, creating multiple users with the :ref:`user module <user_module>`, and
```

Comment:
I would not include the note about `yum`.  As of 2.7 that generates a warning (re: squashing).  The correct way with `yum` is just to provide the list of packages directly to the `name` parameter.

Reply:
Good point - I'll re-work the intro a bit so it's clearer. Can you think of other good, simple, obvious examples of use cases for loops?

Reply:
replaced the `yum` example with a `file` example